packages/@atjson/document/src/collection.ts

```diff
@@ -42,18 +56,8 @@ export class Collection {
     return this.annotations.reduce(reducer, initialValue);
   }
 
-  sort(sortFunction?: (a: Annotation<any>, b: Annotation<any>) => number) {
-    if (sortFunction) {
-      this.annotations = this.annotations.sort(sortFunction);
-    } else {
-      this.annotations = this.annotations.sort((a, b) => {
-        if (a.start - b.start === 0) {
-          return a.end - b.end;
-        } else {
-          return a.start - b.start;
-        }
-      });
-    }
+  sort(sortFunction = compareAnnotations) {
+    this.annotations = this.annotations.sort(sortFunction);
```

Comment:
Whelp, this was previously weird— arrays are sorted in place, so a way to do this in the same manner is to `[...this.annotations].sort(sortFunction)`.